.github/workflows/root-docs-ci.yml

```diff
@@ -212,6 +212,8 @@ jobs:
       run: |
         echo ${RWEBEOS_KT} | base64 -d > ${KT_FILE_NAME}
         kinit -p ${{ secrets.KRB5USER }}@${{ secrets.KRB5REALM }} -kt ${KT_FILE_NAME}
+        EOS_HOST=${EOS_ENDPOINT#root://}
+        xrdfs "${EOS_HOST}" rm -r "${EOS_BASE_PATH}/doc/${WEB_DIR_NAME}" || true
```

Comment:
A standard "trick" for these kind of cases is first copying into a second directory, then swap the two folders (which should be pretty fast), and finally delete the old directory (which might again take some time).